--- strigi-0.7.7/strigidaemon/bin/daemon/eventlistener/eventlistenerqueue.cpp.bero	2012-06-08 17:36:40.714890743 +0200
+++ strigi-0.7.7/strigidaemon/bin/daemon/eventlistener/eventlistenerqueue.cpp	2012-06-08 17:36:59.768890835 +0200
@@ -27,9 +27,7 @@
 #include <iostream>
 #include <stdio.h>
 
-#if defined(__SUNPRO_CC)
-#include <unistd.h>
-#endif
+#include <unistd.h> // for sleep()
 
 using namespace std;
 
